docs/reference/troubleshooting/pepr-istiod-webhooks.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Istiod and Pepr Startup Failures Due to Webhook Dependencies
+
+sidebar:
+  order: 1
+---
+
+## Overview
+During cluster restarts or after upgrades, Istiod and Pepr pods may fail to start properly due to a circular dependency between their admission webhooks.
+
+Both Istiod and Pepr depend on each other’s webhooks being available, which can cause a deadlock if one cannot admit the other’s workloads.
+
+This document explains how to manually resolve the deadlock.
+
+## Symptoms
+- Pepr pods are stuck in CrashLoopBackOff or Pending state.
+- Istiod pods are stuck in CrashLoopBackOff or Pending state.
+- Cluster logs show webhook admission errors like:
```

Comment:
nit: I think this would be `events` not `logs` - likely showing on the replicasets?